package.json

```diff
@@ -76,16 +76,16 @@
   },
   "devDependencies": {
     "chai": "~3.5.0",
-    "chai-as-promised": "^5.3.0",
+    "chai-as-promised": "^6.0.0",
     "chai-jwt": "^2.0.0",
     "coveralls": "^2.11.12",
     "cucumber": "^1.0.0",
     "eslint": "^3.2.2",
     "eslint-config-screwdriver": "^2.0.0",
-    "eslint-plugin-import": "^1.12.0",
-    "github": "^3.0.0",
+    "eslint-plugin-import": "^2.0.0",
+    "github": "^5.1.0",
```

Comment:
https://github.com/mikedeboer/node-github/blob/master/CHANGELOG.md#500 this may break us.